homeassistant/components/flipr/__init__.py

```diff
@@ -0,0 +1,111 @@
+"""The Flipr integration."""
+from datetime import timedelta
+import logging
+
+from flipr_api import FliprAPIRestClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import CONF_FLIPR_IDS, DOMAIN, MANUFACTURER, NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Flipr from a config entry."""
+    _LOGGER.debug("async_setup_entry starting")
+
+    hass.data.setdefault(DOMAIN, {})
+
+    flipr_ids = entry.data[CONF_FLIPR_IDS].split(",")
+
+    # Create all flipr_ids coordinators & devices present in the ConfigEntry
+    for flipr_id in flipr_ids:
+        coordinator = FliprDataUpdateCoordinator(hass, entry, flipr_id)
+        await coordinator.async_config_entry_first_refresh()
+        hass.data[DOMAIN][entry.entry_id + flipr_id] = coordinator
```

Comment:
I think this is cleaner, also on unload you can just pop de entry ID without iterating again.
```suggestion
        hass.data[DOMAIN][entry.entry_id][flipr_id] = coordinator
```

Reply:
I integrated it but it did not work at runtime. I had to had this line before adding coordinators : 
hass.data[DOMAIN][entry.entry_id] = {}